src/sec_tester/repeater.cr

```diff
@@ -23,19 +21,6 @@ module SecTester
         }
       )
       deploy
-      spawn do
-        heartbeat
-      end
-    end
-
-    private def heartbeat
-      loop do
-        sleep 10.seconds
-        break unless @running
-        @socket.emit("ping")
-      end
-    rescue e : Exception
-      Log.error { "Repeater heartbeat error: #{e.message}" }
     end
```

Comment:
We only want to take down this part,and line 26-28. all others are needed.
We still need the `@running` because it is used outside to know if Repeater is up or not.